spec/controllers/protected_resources_controller_spec.rb

```diff
@@ -174,4 +174,106 @@ def doorkeeper_unauthorized_render_options(error: nil)
       end
     end
   end
+
+  context 'when custom forbidden render options are configured' do
+    before do
+      expect(Doorkeeper::AccessToken).to receive(:by_token).with(token_string).and_return(token)
+      expect(token).to receive(:acceptable?).with([:write]).and_return(false)
+    end
+
+    after do
+      module ControllerActions
+        def doorkeeper_forbidden_render_options(*)
+        end
+      end
+    end
+
+    controller do
+      before_filter -> { doorkeeper_authorize! :write }
+
+      include ControllerActions
```

Comment:
Why do we need to define `ControllerActions`? Can we directly inline the method in this `controller do` block?